casaos-fix-docker-api-version/test-script.sh

```
@@ -0,0 +1,1633 @@
+  echo ""
+  echo "=========================================="
+  
+  # Run the fix script
+  if bash ./run.sh; then
+    echo "=========================================="
+    echo ""
+    print_success "Fix script completed"
+  else
+    echo "=========================================="
+    echo ""
+    print_error "Fix script failed with exit code $?"
```

Shellcheck (0.11.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)